pkg/keto/cmd/get.go

```diff
@@ -106,37 +105,36 @@ func getComputePoolCmdFunc(c *cobra.Command, args []string) error {
 		return err
 	}
 
-	name := ""
-	if len(args) == 1 {
-		name = args[0]
+	if len(args) == 0 {
+		args = []string{""}
```

Comment:
I think you don't need to set this. `args` is already an empty slice of strings.